src/installer/BeatPackageCompiler/Properties/launchSettings.json

```diff
@@ -9,6 +9,11 @@
       "commandName": "Project",
       "commandLineArgs": "--package=filebeat-8.11.0-windows-x86_64 -v --keep-temp-files",
       "workingDirectory": "$(SolutionDir)"
+    },
+    "Agent": {
+      "commandName": "Project",
+      "commandLineArgs": "--package=elastic-agent-8.11.1-windows-x86_64 -v --keep-temp-files",
```

Comment:
`BeatPackageCompiler` and `Agent` are 8.11.1 but `Filebeat` is 8.11.0, is that correct?

Reply:
This is just for debugging...